src/d3d9/d3d9_device.cpp

```diff
@@ -1037,6 +1037,13 @@ namespace dxvk {
     if (srcTexInfo->Desc()->Format != dstTexInfo->Desc()->Format)
       return D3DERR_INVALIDCALL;
 
+    D3DSURFACE_DESC srcSurfDesc;
+    D3DSURFACE_DESC dstSurfDesc;
+    src->GetDesc(&srcSurfDesc);
+    dst->GetDesc(&dstSurfDesc);
+    if (srcSurfDesc.Width != dstSurfDesc.Width || srcSurfDesc.Height != dstSurfDesc.Height)
```

Comment:
Just use `src->GetSurfaceExtent() != dst->GetSurfaceExtent()`. No need to get the descs.